script.js

```diff
@@ -132,7 +131,7 @@ async function lookupLockup(near, accountId) {
   } catch (error) {
     console.warn(error);
     return {
-      lockupAccountId: `${lockupAccountId} doesn't exist`,
```

Comment:
How will it look with this change if the account does not have the corresponding lockup?

Reply:
Good eye, Olga, thank you.
I've fixed that here:
https://github.com/near/account-lookup/pull/17/commits/58d51c45f506782c1483322094ed3b5049ef65e8